andersen-homework/src/main/java/com/example/ticket/service/UserService.java

```diff
@@ -0,0 +1,43 @@
+package com.example.ticket.service;
+
+import com.example.ticket.entity.Ticket;
+import com.example.ticket.entity.User;
+import com.example.ticket.enums.TicketType;
+import com.example.ticket.enums.UserStatus;
+import com.example.ticket.repository.TicketRepository;
+import com.example.ticket.repository.UserRepository;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+@Service
+public class UserService {
+
+
+    @Value("${user-update-and-ticket-creation}")
```

Comment:
don't see where user-update-and-ticket-creation value is set

Reply:
added to application.yaml. i have pushed it to resources file